libs/net/services/Actions/ServiceAction.cs

```diff
@@ -4,6 +4,16 @@
 
 namespace TNO.Services.Actions;
 
+/// <summary>
+/// ServiceActionResult enum, encapsulates how a service action completed.
+/// </summary>
+public enum ServiceActionResult
```

Comment:
Move this to a separate file.  Generally only have multiple objects in a single file when it's the only file that access that object.